src/pages/_app.tsx

```diff
@@ -11,6 +12,43 @@ declare global {
   }
 }
 
+const marianne = localFont({
+  src: [
+    {
+      path: "../fonts/subsetted/Marianne-Regular-subset.woff2",
+      weight: "400",
+      style: "normal",
+    },
+    {
+      path: "../fonts/subsetted/Marianne-Regular_Italic-subset.woff2",
+      weight: "400",
+      style: "italic",
+    },
+    {
+      path: "../fonts/subsetted/Marianne-Medium-subset.woff2",
+      weight: "500",
+      style: "normal",
+    },
+    {
+      path: "../fonts/subsetted/Marianne-Bold-subset.woff2",
+      weight: "700",
+      style: "normal",
+    },
+    {
+      path: "../fonts/subsetted/Marianne-ExtraBold-subset.woff2",
+      weight: "800",
+      style: "normal",
+    },
+  ],
```

Comment:
As we define local font for some font-weight and font style, could we get rid of the unused .woff2 and subsetted woff2 files?

Reply:
Yeah I kept them for now in case the design changes a bit while content / design is not that set yet.. Not a huge deal to keep a few files tracked, as you wish really!